SUPPORT.md

```diff
@@ -0,0 +1,4 @@
+# Support
+
+We strive to support all community members and users of devonfw as best as we can, weather that be in their efforts to contribute to the framework directly, or to develop applications based on the framework.  
+Please don't hesitate to contact us via [Microsoft Teams](https://teams.microsoft.com/l/team/19%3af92c481ec30345a28a5434bc530a882a%40thread.skype/conversations?groupId=503df57a-d454-4eec-b3bc-d6d87c7c24f8&tenantId=76a2ae5a-9f00-4f6b-95ed-5d33d77c4d61) / [Yammer](https://www.yammer.com/capgemini.com/#/threads/inGroup?type=in_group&feedId=5030942) or via E-Mail at <icsddevonfwsupport.apps2@capgemini.com>.
```

Comment:
I am not quite sure if we can and should to place our company internal channels in this public place. Providing the CG support Email is fine to me, but the other things need to be discussed with @sjimenez77 and others...

Reply:
I wasn't sure weather I sould include them either, but it seemed to be so little content to just have the CG support e-mail in here (also kind of redundant if we include it in `CODE_OF_CONDUCT.md`).

That's the same problem I'd have with a potential `SECURITY.md`; is it really worth bloating our repos with these default files, if they just contain a short "marketing pitch" and not much specific info?

Reply:
It is not possible for outside Capgemini collaborators to contact us via MS Teams , so from my point of view we should only add email addresses or suggest the corresponding GitHub issue space for each asset.